topics/fair/tutorials/inveniordm-integration/tutorial.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: tutorial_hands_on
+title: Integrating InvenioRDM-compatible Repositories with Galaxy
+tags:
+  - fair
```

Comment:
```suggestion
  - fair
  - InvenioRDM
```

Reply:
Not sure if this is a good tag. But I feel like something more descriptive of the tutorial would be nice.